### DISCUSSION TITLE
Default Port

### DISCUSSION THREAD

**Discussion by eugen257** [mealie-recipes/mealie#3438] [General]
Hi,
please tell me how to change port 9000 to 80. I would like to use this site on a local network and therefore it doesn’t matter to me that it is not safe. I don’t want to create unnecessary services like proxies just to access the site by name only.
Thank you.

**Comment by Kuchenpirat** [comment]
Hi, 
if you are using our [default docker-compose file](https://docs.mealie.io/documentation/getting-started/installation/sqlite/) you would simply have to change the outside port to 80. 

```
    ports:
        - "80:9000"
```

**Comment by eugen257** [comment]
Hi,

I would like it this:

ports:
        - "9000:80"

or better without a port: 80:80

**Reply by Kuchenpirat** [reply]
with ports 9000:80 mealie would be accessible on port 9000 on your machine (as well as it not working, because port inside of the container is set to 80 instead of 9000). So i guess that is not what you want. 

Is there any reason my proposed setup does not meet your requirements?

**Reply by boc-the-git** [reply]
Jumping on Discord might get you better help. Kuchen has accurately answered the questions you've asked.

**Reply by Mraedis** [reply]
To expand on the explanation: mealie is hardcoded inside the container to listen to port 9000, so you cannot change the second number to anything other than 9000 because that's just how mealie works. The first number, which Kuchenpirat has changed to 80 in his solution, is what would be the port that is used on your machine/container host, this you can change to (almost) anything you like.

**Reply by eugen257** [reply]
It turns out that port 9000 cannot be changed. the question is settled

**Comment by eugen257** [comment]
;)) i want so:
Http://mealie.me
Without port

**Comment by hay-kot** [comment]
This is documented.

![CleanShot 2024-04-08 at 10 12 40@2x](https://github.com/mealie-recipes/mealie/assets/64056131/6bb53732-ce61-4711-9b84-40e3f130083c)

---

However, if you are running in docker. You do not want to do this. What you actually want to do is _map_ the container port to the host port of your choosing. That is what the others were recommending.

Changing the docker compose mappings to 

```
80:9000
```

Will map the containers port `9000` to the hosts port `80` (in most configurations). If you are running this directly on the host machine, then you would need to modify the `API_PORT` env variable.

Then if your dns is setup correctly, you should be able to access it with http://mealie.me/ like you're wanting to.
